### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by LaskoA**
Please check my solution and help to fix failure related to "test_create_movie" in class AdminMovieAPITests.

Thank you.

### CODE REVIEW

File: cinema/views.py [hunk 1]
```diff
@@ -126,6 +123,28 @@ def upload_image(self, request, pk=None):
...
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "title",
+                type={"type": "list", "items": {"type": "str"}},
+                description="Filter by title",
+            ),
+            OpenApiParameter(
+                "genres",
+                type={"type": "list", "items": {"type": "number"}},
```
Comment (Abnormaltype) [inline cinema/views.py:135]: the same here

File: cinema/views.py [hunk 2]
```diff
@@ -126,6 +123,28 @@ def upload_image(self, request, pk=None):
...
+                description="Filter by genre.id",
+            ),
+            OpenApiParameter(
+                "actors",
+                type={"type": "list", "items": {"type": "number"}},
```
Comment (Abnormaltype) [inline cinema/views.py:140]: the saem here

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
Please check now, thank you :)

**Review by IvanRamyk — APPROVED** [review]
Good job!

**Review by Abnormaltype — APPROVED** [review]

**Pull request closed without merging**